src/main/java/bio/terra/cli/command/spend/Enable.java

```diff
@@ -17,15 +19,20 @@ public class Enable extends BaseCommand {
 
   /** Print this command's output in text format. */
   private static void printText(UFSpendProfileUser returnValue) {
-    OUT.println("User enabled on the default spend profile.");
+    OUT.println("User enabled on the spend profile.");
```

Comment:
print the specific profile here (like with disable)?

Reply:
It's being printed as part of the UFSpendProfileUser.print(). For ex:
```
> terra spend enable --email=ishaghag@google.com --policy=user --profile=wm-alt-spend-profile

User enabled on the spend profile.
ishaghag@google.com: wm-alt-spend-profile(OWNER, USER)
```

Reply:
sg